run.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 set -e
 
-## commented out for now until further work on lxc
-#/patchcontainer.sh
-#brctl addbr lxcbr0
-#ifconfig lxcbr0 10.0.4.1 up
+# This script is run if the jujubox image is run without any arguments.
 
 export HOME=/home/ubuntu
 cd $HOME
-chown ubuntu:ubuntu $HOME/.juju
 sudo -u ubuntu ssh-agent /bin/bash
```

Comment:
Agree on this, I want to make that a separate pull request for that functionality.